staticaddr/loopin/sql_store.go

```diff
@@ -0,0 +1,336 @@
+package loopin
```

Comment:
nit: This could be a separate commit.

Reply:
+1

Reply:
done

Reply:
Separated `loopin.go` and `sql_store.go` into their own commits.